ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -63,17 +63,14 @@ export const ImportNftsModal = ({ onClose }) => {
   const nftsDropdownState = useSelector(getNftsDropdownState);
   const selectedAddress = useSelector(getSelectedAddress);
   const chainId = useSelector(getCurrentChainId);
-  const addressEnteredOnImportTokensPage =
-    history?.location?.state?.addressEnteredOnImportTokensPage;
-  const contractAddressToConvertFromTokenToNft =
-    history?.location?.state?.tokenAddress;
+  const {
+    tokenAddress: initialTokenAddress,
+    tokenId: initialTokenId,
+    ignoreErc20Token,
+  } = useSelector((state) => state.appState.importNftsModal);
```

Comment:
the function `(state) => state.appState.importNftsModal)` can be extracted to the file selectors.js and be imported , like what we're doing for the line above

Reply:
My take is that a function isn't needed for one line that's called in 1 place.  I think it adds more code to maintain without benefit.  I *would* do a function if there was logic to encapsulate, or it was reused in several places.